WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -137,21 +136,34 @@ private extension SupportTableViewController {
 
     // MARK: - Table Model
 
-    func wordPressAppTableViewModel() -> ImmuTable {
-
-        // Community Forums Section
-        var communityForumsSectionRows = [ImmuTableRow]()
-        communityForumsSectionRows.append(SupportForumRow(title: LocalizedText.wpForumPrompt,
-                                                          action: nil,
-                                                          accessibilityIdentifier: "visit-wordpress-forums-prompt"))
-        communityForumsSectionRows.append(SupportForumButtonRow(title: LocalizedText.visitWpForumsButton,
-                                                                accessibilityHint: LocalizedText.visitWpForumsButtonAccessibilityHint,
-                                                                action: visitForumsSelected(),
-                                                                accessibilityIdentifier: "visit-wordpress-forums-button"))
-
-        let forumsSection = ImmuTableSection(headerText: LocalizedText.wpForumsSectionHeader,
-                                             rows: communityForumsSectionRows,
-                                             footerText: nil)
+    func tableViewModel() -> ImmuTable {
```

Comment:
Small nit: We could consider passing `SupportConfiguration.current()` into this function.

Reply:
Good idea, it makes it less reliant on globals. I made this change now.